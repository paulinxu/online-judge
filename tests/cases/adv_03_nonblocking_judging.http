###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0}
###
Got error: reqwest::Error { kind: Request, url: Url { scheme: "http", cannot_be_a_base: false, username: "", password: None, host: Some(Ipv4(127.0.0.1)), port: Some(12345), path: "/jobs", query: None, fragment: None }, source: TimedOut }
