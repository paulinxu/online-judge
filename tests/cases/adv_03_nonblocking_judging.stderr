[2024-07-25T15:32:27Z INFO  actix_server::builder] starting 16 workers
[2024-07-25T15:32:27Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2024-07-25T15:32:28Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000229
[2024-07-25T15:32:31Z INFO  oj] Shutdown as requested
